test/datahike/test/migrate_test.clj

```diff
@@ -117,9 +117,9 @@
         (let [cfg (merge base-config
                          {:keep-history? hist
                           :attribute-refs? attr-ref})
-              old-cfg (assoc-in cfg [:store :path] old-path)
+              old-cfg (assoc-in cfg [:store :path] (str old-path (.getTime (java.util.Date.))))
```

Comment:
It seems to me like it would make sense to have a little helper function that creates ids based on time. This would make it more explicit what this string is there for as the usage of date objects makes it a bit confusing.

I would also suggest to always create a function in `datahike.tools` or even in a special compatibility namespace when java functions are used in order to facilitate the upkeep of clojurescript compliance. For the creation of a date object, there is already a function currently named `get-time` in `datahike.tools` that works with clj and cljs.

Reply:
Yes, I built konserve, kabel, replikativ etc. that way, where we have code in a `platform` namespace for each platform. This is fixed now.